.github/CONTRIBUTING.md

```diff
@@ -99,9 +101,30 @@ The previous code made compliant:
 /datum/datum1/datum2/proc/proc3()
 	code
 /datum/datum1/datum2/proc2()
+	. = ..()
+	code
+```
+
+### Preserve return values
+```
+/datum/proc/proc1()
+	code
+	return something
+
+/datum/datum1/proc1()
 	..()
+```
+This wouldn't preserve the return value of the parent proc, instead, you should do this.
+```
+/datum/proc/proc1()
 	code
+	return something
+
+/datum/datum1/proc1()
+	. = ..()
 ```
+`.` is the current return value of the current proc.
+If you have the parent call at the end of the proc, use `return ..()` instead, it's slightly faster.
```

Comment:
The speed isnt really why we do it, it's mostly just been maintainer preference in the past because they don't like the look of `. = ..()` as the last line